fastlane/Fastfile

```diff
@@ -179,6 +198,44 @@ platform :ios do
     )
   end
 
+  #####################################################################################
+  # update_jetpack_appstore_strings
+  # -----------------------------------------------------------------------------------
+  # This lane updates the AppStoreStrings.po files for the WordPress app with
+  # the latest content from the release_notes.txt file and the other text
+  # sources
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_jetpack_appstore_strings version:<release note version>
+  #
+  # Example:
+  # bundle exec fastlane update_jetpack_appstore_strings version:10.7
+  #####################################################################################
+  desc 'Updates the AppStoreStrings.po file for the WordPress app with the latest data'
+  lane :update_jetpack_appstore_strings do |options|
+    source_metadata_folder = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'appstoreres', 'jetpack_metadata', 'source')
+
+    files = {
+      whats_new: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt'),
+      app_store_subtitle: File.join(source_metadata_folder, 'subtitle.txt'),
+      app_store_desc: File.join(source_metadata_folder, 'description.txt'),
+      app_store_keywords: File.join(source_metadata_folder, 'keywords.txt'),
+      'screenshot-text-1' => File.join(source_metadata_folder, 'promo_screenshot_1.txt'),
+      'screenshot-text-2' => File.join(source_metadata_folder, 'promo_screenshot_2.txt'),
+      'screenshot-text-3' => File.join(source_metadata_folder, 'promo_screenshot_3.txt'),
+      'screenshot-text-4' => File.join(source_metadata_folder, 'promo_screenshot_4.txt'),
+      'screenshot-text-5' => File.join(source_metadata_folder, 'promo_screenshot_5.txt'),
+      'screenshot-text-6' => File.join(source_metadata_folder, 'promo_screenshot_6.txt'),
+      'app-store-name' => File.join(source_metadata_folder, 'app_name.txt')
+    }
+
+    ios_update_metadata_source(
+      po_file_path: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'AppStoreStrings.po'),
+      source_files: files,
+      release_version: options[:version]
+    )
+  end
```

Comment:
There's admittedly some duplication between this and the WordPress lane. I think we can live with that for the moment and think about removing it later if needed.

Reply:
Agreed – copy/paste is our friend here – no point over-abstracting